xeth/whisper.go

```diff
@@ -36,7 +36,7 @@ func (self *Whisper) Post(payload string, to, from string, topics []string, prio
 			TTL:    time.Duration(ttl) * time.Second,
 			To:     crypto.ToECDSAPub(common.FromHex(to)),
 			From:   key,
-			Topics: whisper.TopicsFromString(topics...),
+			Topics: whisper.NewTopicsFromStrings(topics...),
```

Comment:
Actually, not...

If we have `NewTopic`, then `NewTopicString` makes more sense, then `NewTopic` combined with `TopicFromString`. Since NewTopicsStrings sounds a bit weird, I'd stick to the currently proposed version of NewTopicsFromStrings.